app/screens/settings/notifications/notifications.tsx

```diff
@@ -55,6 +59,17 @@ const Notifications = ({isCRTEnabled, enableAutoResponder}: NotificationsProps)
         );
     };
 
+    const goToNotificationSettingsMentions = () => {
+        const screen = Screens.SETTINGS_NOTIFICATION_MENTION;
+
+        const id = isCRTEnabled ? t('notification_settings.mentions') : t('notification_settings.mentions_replies');
+        const defaultMessage = isCRTEnabled ? 'Mentions' : 'Mentions and Replies';
+        const title = intl.formatMessage({id, defaultMessage});
+        const passProps = {isCRTEnabled};
```

Comment:
0/5 Shouldn't the screen we are opening be the one observing whether CRT is enabled?
I could understand we don't, but just making sure.